Azure/azure-cli
# [AppService] Bugfix: az webapp up fails when user doesn't have write permissions to project's parent directory

**Description**  
Fixes #15199 
`az webapp up` zips the project folder to zipdeploy then deletes it after. If the user doesn't have write permissions to create a zip, it fails
Instead of creating zip in the same folder as the project, create in temp directory where user will have permissions to

**Testing Guide**  
- Remove write permissions to the folder where the project you want to deploy is
- Open cmd/terminal/shell without administrator privileges
- Run `az webapp up...` to deploy project
- Fails due to permission denied

---

This checklist is used to make sure that common guidelines for a pull request are followed.
